src/App.vue

```diff
@@ -1,15 +1,24 @@
 <template>
   <div id="app">
+    <HeaderLayout />
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts">
+import HeaderLayout from "./presentation/layout/HeaderLayout.vue";
 export default {
   name: "App",
+  components: {HeaderLayout},
 }
 </script>
 
-<style scoped>
-
-</style>
\ No newline at end of file
+<style>
+  * {
```

Comment:
use normalise.css instead

Reply:
As i understand, it is done

Reply:
so why does it still here?

Reply:
about * {box-sizing: border-box} https://github.com/necolas/normalize.css/issues/202